app/models/user/time/TimeSpan.scala

```diff
@@ -124,24 +120,24 @@ class TimeSpanDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
     Json.toJson(tuples.map(formatTimespanTuple))
   }
 
-  def findAllByAnnotation(annotationId: ObjectId, start: Option[Long], end: Option[Long]): Fox[List[TimeSpan]] =
+  def findAllByAnnotation(annotationId: ObjectId, start: Option[Instant], end: Option[Instant]): Fox[List[TimeSpan]] =
     for {
       r <- run(
         Timespans
           .filter(
             r =>
-              notdel(r) && r._Annotation === annotationId.id && (r.created >= new java.sql.Timestamp(
-                start.getOrElse(0))) && r.created <= new java.sql.Timestamp(end.getOrElse(MAX_TIMESTAMP)))
+              notdel(r) && r._Annotation === annotationId.id && (r.created >= start
+                .getOrElse(Instant(0))
```

Comment:
The formatting seems a bit weird to me here, since `.getOrElse` is only about `start`. But no strong opinion.

Reply:
Yeah, that’s what our formatter does here. I think mid-term, we’d want to replace this kind of query with the sql syntax ones, then we don’t have this problem anymore